assets/css/custom.scss

```
@@ -3908,3 +3908,1439 @@ font-size: 1.55rem;
+      border-bottom: 1px solid rgba(255, 255, 255, 0.07);
+      &:last-child {
+        border-bottom: 0;
+      }
+    }
+
+    &__value {
+      font-size: 2.2rem;
+      line-height: 1.1;
+      font-weight: 900;
+      @include cs-brand-gradient-text;
+      letter-spacing: -0.01em;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)